docs/documentation/functions/certificates/Invoke-GeneratePrivateKeyAndCsr.md

```diff
@@ -0,0 +1,193 @@
+# Invoke-GeneratePrivateKeyAndCsr
+
+## Synopsis
+
+Generate Private Key and certificate signing request (CSR) files using Openssl with a common name and neccesary parameters.
+
+## Syntax
+
+```powershell
+Invoke-GeneratePrivateKeyAndCsr [-certDirPath] <String> [-subjectAlternativeNames] <String> [-keySize] <Int>
+ [-expireDays] <Int> [-organization] <String> [-organizationUnit] <String> [-locality] <String>
+ [-state] <String> [-country] <String>
+```
+
+## Description
+
+The `Invoke-GeneratePrivateKeyAndCsr` cmdlet generates the private key and certificate signing request (CSR) files using Openssl with a common name and neccesary parameters.
```

Comment:
```suggestion
The `Invoke-GeneratePrivateKeyAndCsr` cmdlet generates the private key and certificate signing request (CSR) files using OpenSSL with a common name and neccesary parameters.
```

Reply:
addressed in commit cfcd77a79eb20278e44b123b0092c173dd7b5014